src/core.jl

```diff
@@ -429,10 +431,31 @@ function get_def_graph()
 end
 has_def_graph() = isdefined(def_graph, :x)
 
+"""
+Sets the default computation graph to `g`.
```

Comment:
Can you include the function signature at the top of the docstring, as seemed to be conventional?